packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -2893,10 +2945,13 @@ class _MenuItemLabel extends StatelessWidget {
           ),
       ],
     );
-    if (semanticsLabel != null) {
+    if (semanticsLabel != null || isEnabled != null || isSemanticButton != null) {
       menuItemLabel = Semantics(
         label: semanticsLabel,
-        excludeSemantics: true,
+        enabled: isEnabled,
+        button: isSemanticButton,
```

Comment:
we should programatically assign the button: true based on defaultTargetPlatform, this is not something developer can easily pick and know about the consequence, and it seems  like there may not be a good reason to provide the flexibility